packages/fxa-auth-server/lib/routes/validators.js

```diff
@@ -416,13 +416,27 @@ module.exports.subscriptionsPlaySubscriptionSupportValidator = isA
   })
   .unknown(true);
 
+module.exports.subscriptionsAppStoreSubscriptionSupportValidator = isA.object({
```

Comment:
Should we add `price_id` to this?

At the moment type `AppStoreSubscription` requires `price_id` (via `IapExtraStripeInfo`) but requests to `/oauth/support-panel/subscriptions` fail because of this validation.

I believe this should have been fixed by #13328 , so you might just need to rebase onto `main`.

Reply:
Thanks Reino -- yes I just merged that last night and pinged Meghan about it -- also see [this comment in the Jira ticket](https://mozilla-hub.atlassian.net/browse/FXA-5227?focusedCommentId=560270).

Reply:
Rebased yesterday and added updates to files! Thank you, Reino & Bianca! ❤️